tests/python-pytest/onnx/export/mxnet_export_test.py

```diff
@@ -179,8 +180,80 @@ def test_model_accuracy(model_name, input_shape):
         npt.assert_equal(expected.shape, actual.shape)
         npt.assert_almost_equal(expected, actual, decimal=3)
 
+def test_greater():
+    """Test for logical greater in onnx operators."""
+    input1 = np.array([[ 1,  1,  1], [ 1,  1,  1]]).astype("float32")
+    input2 = np.array([[ 0], [ 1]]).astype("float32")
+    op = np.array([[ 1,  1,  1], [ 0,  0,  0]]).astype("float32")
```

Comment:
do not hard code this operation. use `np.greater`. Also can you please name this variable different, like `expected_value` or `numpy_op` or something like that.